WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationFormViewModel.swift

```diff
@@ -355,6 +356,11 @@ final class BlazeCampaignCreationFormViewModel: ObservableObject {
                 ctaText = firstSuggestion.ctaText
             }
         } catch {
+            if let name = product?.name, let shortDescription = product?.shortDescription {
+                tagline = name
+                description = shortDescription
+                ctaText = Localization.shopNow
+            }
```

Comment:
Should this be the other way around? I have a product without short description and the pre-populated tagline makes it look odd:

<img src="https://github.com/user-attachments/assets/4959b047-ba4d-4298-99a9-6e1b82aee7e8" width=320 />

Reply:
Thanks for the explanation. Maybe @JorgeMucientes can confirm the requirements?
Either way, I think we should display the description when short description is not available.

Reply:
I just checked the "Edit Ad" screen, and I think it should remain this way. In fact, the `tagline` has a limited number of characters, which is fewer than the number of characters allowed in the `description`.
I agree on displaying the description, if short description is not available: 38e93e148e6423131fe252495b2c4247664d15e4